app/apollo/test/cluster.spec.js

```diff
@@ -944,7 +944,10 @@ describe('cluster graphql test suite', () => {
         orgId: org01._id,
         registration: { name: 'my-cluster-plus-1' },
       });
-
+      registerCluster = await clusterApi.registerCluster(adminToken, {
+        orgId: org01._id,
+        registration: { name: 'my-cluster-plus-2' },
+      });
```

Comment:
Why is this change neccessary when it was not previously needed?  It's a non-obvious change -- if it's justified, code comments should be added to explain what is happening and why it's needed.

Reply:
It is no longer needed after other changes. I originally added it due to changing the max_number of clusters in travis.yml so it would still pass it's test. Reverting this change.